nillion_client_script.py

```diff
@@ -0,0 +1,161 @@
+bootnodes = [bootnode]
+
+async def store_inputs_and_run_blind_computation(input_data, program_name, output_parties, nilchain_private_key):
+    # Create Nillion Client for user
+    seed=str(uuid.uuid4())
+    userkey = UserKey.from_seed(f"nada-by-example-{seed}")
+
+    nodekey = NodeKey.from_seed(seed)
+    client = create_nillion_client(userkey, nodekey, bootnodes)
+
+    party_id = client.party_id
+    user_id = client.user_id
+
+    # Pay for and store the program
+    # Set the program name and path to the compiled program
+    program_mir_path = f"./target/{program_name}.nada.bin"
+    # Create payments config, client and wallet
+    payments_config = create_payments_config(chain_id, grpc_endpoint)
+    payments_client = LedgerClient(payments_config)
+
+    # Check that private key is set in the streamlit secrets file
+    try:
+        private_key = PrivateKey(bytes.fromhex(nilchain_private_key))
+        print("Using Nilchain private key...")
+    except Exception as e:
+        raise RuntimeError(f"Invalid Nilchain private key! Before running blind computation, set your Nilchain Testnet nilchain_private_key within the .streamlit/secrets.toml secrets file.")
+    
+    payments_wallet = LocalWallet(
+        private_key,
+        prefix="nillion",
+    )
+
+    # Pay to store the program and obtain a receipt of the payment
+    memo_store_program = f"petnet operation: store_program; program_name: {program_name}; user_id: {user_id}"
+    receipt_store_program = await get_quote_and_pay(
+        client,
+        nillion.Operation.store_program(program_mir_path),
+        payments_wallet,
+        payments_client,
+        cluster_id,
+        memo_store_program,
+    )
+    print(f"🧾 RECEIPT MEMO: {memo_store_program}")
+
+    # Store the program
+    program_id = await client.store_program(
+        cluster_id, program_name, program_mir_path, receipt_store_program
+    )
+
+    print(program_id)
+
+    compute_bindings = nillion.ProgramBindings(program_id)
+
+    for party_name in output_parties:
+        compute_bindings.add_output_party(party_name, party_id)
+
+    store_ids=[]
+    # Store each input
+    for input_name, details in input_data.items():
+        value, party_name, input_type = details
+        print(f"Processing Input: {input_name}, Value: {value}, Party: {party_name}, Type: {input_type}")
+        
+        
+        if input_type == 'SecretInteger':
+            nada_val = nillion.SecretInteger(value)
+        elif input_type == 'SecretBoolean':
+            nada_val = nillion.SecretBoolean(value)
+        elif input_type == 'PublicBoolean':
+            nada_val = nillion.Boolean(value)
+        elif input_type == 'SecretUnsignedInteger':
+            nada_val = nillion.SecretUnsignedInteger(value)
+        elif input_type == 'PublicUnsignedInteger':
+            nada_val = nillion.UnsignedInteger(value)
+        else:
+            nada_val = nillion.Integer(value)
```

Comment:
Nit but might make it easier to maintain:
```python
# Nada types to Nillion types
types_mapping = {
    # Integers
    'SecretInteger': nillion.SecretInteger,
    'PublicInteger': nillion.Integer,
    # Unsigned Integers
    'SecretUnsignedInteger': nillion.SecretUnsignedInteger,
    'PublicUnsignedInteger': nillion.UnsignedInteger,
    # Booleans
    'SecretBoolean': nillion.SecretBoolean,
    'PublicBoolean': nillion.Boolean,
}

nada_val = types_mapping.get(input_type, nillion.Integer)(value)
```

I'm also not sure if we need to use `nillion.Integer` as the default if we don't find it in the map, maybe it'd be better to throw an error?